backend/app/Http/Requests/AddWebsiteHttpRequest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+declare(strict_types=1);
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+final class AddWebsiteHttpRequest extends FormRequest
+{
+    public function authorize(): bool
+    {
+        return true;
+    }
+
+    public function rules(): array
+    {
+        return [
+            'name' => 'required|string|max:255',
+            'domain' => 'required|string|max:255',
+            'single_page' => 'boolean',
+        ];
+    }
+
+    public function name(): string
+    {
+        return $this->get('name');
+    }
+
+    public function domain(): string
+    {
+        return $this->get('domain');
+    }
+
+    public function singlePage(): ?bool
```

Comment:
it cannot be null as you validate it by boolean

Reply:
Fixed.